scripts/image_downloads_query.sh

```diff
@@ -1,6 +1,6 @@
 VERSION=$1
 
-image_shas=$(aws ecr describe-images --registry-id 906394416424 --repository-name aws-for-fluent-bit --region us-west-2 --image-ids imageTag=arm64-${VERSION} imageTag=amd64-${VERSION} imageTag=init-arm64-${VERSION} imageTag=init-amd64-${VERSION} imageTag=${VERSION}-ltsc2019 imageTag=${VERSION}-ltsc2022 | jq -r '.imageDetails[] | objects | .imageDigest')
+image_shas=$(aws ecr describe-images --registry-id 906394416424 --repository-name aws-for-fluent-bit --region us-west-2 --image-ids imageTag=${VERSION} imageTag=arm64-${VERSION} imageTag=amd64-${VERSION} imageTag=init-arm64-${VERSION} imageTag=init-amd64-${VERSION} imageTag=${VERSION}-windowsservercore imageTag=${VERSION}-ltsc2019 imageTag=${VERSION}-ltsc2022 | jq -r '.imageDetails[] | objects | .imageDigest')
```

Comment:
We have two  imageTag=arm64-${VERSION} and two imageTag=init-arm64-${VERSION}. Is this correct?

Reply:
Yes it is correct. I only see each once. I have checked that the output is correct:

```
147dda285fd4:aws-for-fluent-bit wppttt$ bash scripts/image_downloads_query.sh 2.31.11
Paste below query into Athena console and replace eventTime with desired data range
This can be used to obtain 'private' ECR download count
The query is for classic regions but can be used in other regions by changing the table name to 'cloudtrail_fluentbit'
-------


SELECT COUNT(*) as call_count
FROM cloudtrail_fluentbitimage
WHERE
    eventsource = 'ecr.amazonaws.com' AND
    eventname in ('GetDownloadUrlForLayer', 'BatchGetImage') AND
    eventTime > 'YYYY-MM-DDT00:00:00Z' AND
    eventTime < 'YYYY-MM-DDT00:00:00Z' AND (
      requestparameters like '%imageTag:2.31.11%' OR
      requestparameters like '%imageTag:amd64-2.31.11%' OR
      requestparameters like '%imageTag:arm64-2.31.11%' OR
      requestparameters like '%imageTag:init-2.31.11%' OR
      requestparameters like '%imageTag:init-amd64-2.31.11%' OR
      requestparameters like '%imageTag:init-arm64-2.31.11%' OR
      requestparameters like '%imageTag:2.31.11-windowsservercore%' OR
      requestparameters like '%imageTag:2.31.11-ltsc2019%' OR
      requestparameters like '%imageTag:2.31.11-ltsc2022%' OR
      requestparameters like '%sha256:ff702d8e4a0a9c34d933ce41436e570eb340f56a08a2bc57b2d052350bfbc05d%' OR
      requestparameters like '%sha256:8e65a27f585e363e68ad0d5bd75cdb7dee13b945bf267c1238c99e337af11d3d%' OR
      requestparameters like '%sha256:31d7a29391a7093f6f2c357e36343c58600be21fcb90bd78b0f3ab8ceec97671%' OR
      requestparameters like '%sha256:3537ce0a2f3906e4837fd7c99ccd429d1d42706188c005ca65a24a8de719c4ca%' OR
      requestparameters like '%sha256:dd240cd81d6798de36c14a46fad9a378f19823ba63ecdde3c97bd2e250a9ac08%' OR
      requestparameters like '%sha256:d6184d9d355a7c2392a81cc0af0ade1eb84740b2eff1a2be37fba2151b16bc43%' OR
      requestparameters like '%sha256:3046164456953cc274804643ed9cd82c064b9e2894d5fbb42d101909199ac227%' OR
      requestparameters like '%sha256:b9f1f121369f8b8ba17b64fab8e38897819156eb0d84d8025ada6e939be55e67%'
    )
```